### PR TITLE
feat: add scrollbar-* properties

### PR DESCRIPTION
**Pull request by redonkulus**
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

---

Requested by @nspector. 

Adding `scrollbar-*` properties

### CODE REVIEW

File: packages/atomizer/src/rules.js [hunk 1]
```diff
@@ -3300,6 +3300,74 @@ module.exports = [
...
+        styles: {
+            'scrollbar-gutter': '$0 $1',
+        },
+        arguments: [
+            {
+                a: 'auto',
+                s: 'stable',
+            },
+            {
+                be: 'both-edges',
```
Comment (redonkulus) [inline packages/atomizer/src/rules.js:3344]: @carriemorrison I'm not sure if this `both-edges` value applies to both `auto` and `stable` or just `stable`. If its the latter, I could just modify this rule to support a `sbe` value which maps to "stable both-edges".

The spec for reference: https://www.w3.org/TR/2021/WD-css-overflow-3-20211223/#propdef-scrollbar-gutter

**Comment by carriemorrison** [comment]
Screen Shot 2022-12-02 at 4 18 51 PM

**Comment by carriemorrison** [comment]
Same for https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color

**Comment by carriemorrison** [comment]
Maybe leave in draft form till fully adopted?

**Review by nspector — APPROVED** [review]

**Comment by redonkulus** [comment]
Good point about the draft status of `scrollbar-gutter` and `scrollbar-color`. Generally, we only add properties once the reach recommendation status. It looks like there is a ticket open to add to webkit but it hasn't gained traction: https://bugs.webkit.org/show_bug.cgi?id=167335

**Comment by redonkulus** [comment]
Given these rules are not recommended standards yet, let's hold on including them. At least until Safai supports them too.

**Pull request closed without merging**